tox.ini

```diff
@@ -5,6 +5,7 @@ envlist = py, lint, fmt
 description = run unit tests
 deps =
     pytest>=7
+    setuptools
```

Comment:
Is setuptools needed to run unittests?

Reply:
On my pc it is required, but I am not sure if its required in the build. Hence removing it for now.